<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TAB">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="basic_tab">
       <property name="toolTip">
        <string/>
       </property>
       <attribute name="title">
        <string>Basic</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2" colspan="3">
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>20</height>
             </size>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" rowspan="3" colspan="2">
           <widget class="QListWidget" name="listWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>330</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid rgb(3, 3, 3);
border-radius: 5px;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="btn_banner">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color:rgb(58, 134, 255);</string>
            </property>
            <property name="text">
             <string>스캔시작</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_ping">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color:rgb(15, 170, 170)</string>
            </property>
            <property name="text">
             <string>스캔시작</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_stealth">
            <property name="title">
             <string>Stealth 스캔</string>
            </property>
            <widget class="QComboBox" name="combo_stealth">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>20</y>
               <width>151</width>
               <height>22</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(230, 230, 230);</string>
             </property>
             <item>
              <property name="text">
               <string>Half Scan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Xmas Scan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fin Scan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Null Scan</string>
              </property>
             </item>
            </widget>
            <widget class="QLabel" name="lbl_stealth_ip">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>50</y>
               <width>21</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
            <widget class="QLabel" name="lbl_stealth_port">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>90</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_stealth_ip">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>49</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_stealth_port">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>90</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QGroupBox" name="groupBox_banner">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>배너 그랩</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="btn_ack">
            <property name="styleSheet">
             <string notr="true">color: white;
background-color:rgb(91, 170, 148)</string>
            </property>
            <property name="text">
             <string>스캔시작</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_stealth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color:rgb(170, 170, 255);</string>
            </property>
            <property name="text">
             <string>스캔시작</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_ping">
            <property name="title">
             <string>ping 스캔</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2" colspan="3">
           <widget class="QLabel" name="status_change">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(58, 134, 255);
background-color: white;
border: 2px solid rgb(58, 134, 255);
border-radius: 5px;</string>
            </property>
            <property name="text">
             <string>정상 상태</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="2" colspan="3">
           <widget class="QScrollArea" name="output_small">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>220</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_6">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>415</width>
               <height>218</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QGroupBox" name="groupBox_ack">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>ACK 스캔</string>
            </property>
            <widget class="QLabel" name="lbl_stealth_ip_2">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>50</y>
               <width>21</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_ack_ip">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>50</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="lbl_stealth_port_2">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>90</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="input_ack_port">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>90</y>
               <width>231</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_reset">
            <property name="styleSheet">
             <string notr="true">color: white;
background-color:rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>리셋</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advanced_tab">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>get_Stealth()</slot>
 </slots>
</ui>
